graph.py

```diff
@@ -1,5 +1,17 @@
-import networkx as nx
-from networkx.exception import NodeNotFound, NetworkXNoPath
+import subprocess
+import sys
+from kytos.core import log
+
+try:
+    import networkx as nx
+except ImportError:
+    package = 'networkx>=2.2'
+    ret = subprocess.call([sys.executable, "-m", "pip", "install", package])
```

Comment:
This, should not be here.

I didn't get the problem here. What is the problem with pip3 install -r requirments.txt ?

For me you should just add networkx>=2.2 on requirements.txt and update readme procedures to install.

Reply:
@beraldoleal, 

Indeed, networkx>=2.2 should definitely be there. I can drop the rest of the PR and only keep this part. The main problem though, putting yourself in the shoes of the end user, how could the end user know if `kytos/pathfinder` (which depends on networkx) is really a dependency of `kytos/mef_eline`? I bet most users will discover this once they `kytos napps install kytos/mef_eline` and see the depency showing up in the CLI. If you read the current readme of mef_eline pip isn't mentioned for example https://github.com/kytos/mef_eline. So I usually stumble upon this problem and remember to keep reinstalling networkx, I figured auto install would make things more plug and play for the end user.

Reply:
@viniarck at first it can look like a handy thing to do for users, but it's not considered a good practice to insert a `subprocess.call('pip install')` on the top of a Python file. It's a side effect that's not expected by others programmers.

We can keep the exception handling, the log message, and even add the "pip install" line to the log message. The documentation can also be updated.

For a more streamlined experience, we can implement a more general solution where `kytos napps install` also installs any Python libs that the NApp depends on.